config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -11,6 +11,7 @@ en:
             THIS ACTION IS IRREVERSIBLE
             
             Are you sure you want to submit? You will no longer be able to amend your submission!
+          confirm_unsubmit: 'Are you sure you want to unsubmit?'
```

Comment:
That's true, I am not sure if we should allow that and submitted_at time changed. 

At least you should tell what does the button does, can use finalize or reset answer message as an example.